custom/gcc_sangath/tests/test_custom_actions.py

```diff
@@ -0,0 +1,151 @@
+            {
+                'agg_mean_general_skills_score': '0.0',
+                'agg_mean_treatment_specific_score': '0.0',
+                'agg_rating': '0.0',
+                'date_of_peer_review': '',
+                'feedback_num': '1',
+                'share_score_check': 'yes',
+                'total_session_rating': '0'
+            }
+        )
+
+    def test_with_peer_rating_cases(self):
+        rule = create_empty_rule(
+            self.domain, AutomaticUpdateRule.WORKFLOW_CASE_UPDATE, case_type=SESSION_CASE_TYPE,
+        )
+        self._set_up_cases([
+            {
+                MEAN_GENERAL_SKILLS_SCORE_CASE_PROP: 1,
+                MEAN_TREATMENT_SPECIFIC_SCORE_CASE_PROP: 2,
+                SESSION_RATING_CASE_PROP: 3,
+                DATE_OF_PEER_REVIEW_CASE_PROP: date(2020, 1, 1)
+            },
+            {
+                MEAN_GENERAL_SKILLS_SCORE_CASE_PROP: 2,
+                MEAN_TREATMENT_SPECIFIC_SCORE_CASE_PROP: 3,
+                SESSION_RATING_CASE_PROP: 5,
+                DATE_OF_PEER_REVIEW_CASE_PROP: date(2020, 8, 10)
+            },
+            {
+                MEAN_GENERAL_SKILLS_SCORE_CASE_PROP: 1,
+                MEAN_TREATMENT_SPECIFIC_SCORE_CASE_PROP: 2,
+                SESSION_RATING_CASE_PROP: 4,
+                DATE_OF_PEER_REVIEW_CASE_PROP: date(2020, 3, 10)
+            }
+        ])
+
+        result = sanitize_session_peer_rating(self.session_case, rule)
+
+        self.assertEqual(result.num_updates, 1)
+        session_case = CommCareCase.objects.get_case(self.session_case.case_id, self.session_case.domain)
+        self.assertDictEqual(
+            session_case.case_json,
+            {
+                'agg_mean_general_skills_score': '1.3',
+                'agg_mean_treatment_specific_score': '2.3',
+                'agg_rating': '4.0',
+                'date_of_peer_review': '2020-08-10',
+                'feedback_num': '3',
+                'share_score_check': 'yes',
+                'total_session_rating': '12'
+            }
+        )
+
+    def _set_up_cases(self, peer_ratings_case_json=None):
+        peer_ratings_case_json = peer_ratings_case_json or []
+        session_case = CaseStructure(
+            attrs={
+                'create': True,
+                'case_type': SESSION_CASE_TYPE,
+                'update': {
+                    "name": "Session 1",
+                },
+            })
+        peer_rating_cases = []
+        for peer_rating_case_json in peer_ratings_case_json:
+            case_json = {'name': "Peer Rating"}
+            case_json.update(peer_rating_case_json)
+            peer_rating_cases.append(CaseStructure(
+                attrs={
+                    'create': True,
+                    'case_type': PEER_RATING_CASE_TYPE,
+                    'update': case_json,
+                },
+                indices=[CaseIndex(
+                    session_case,
+                    identifier='parent',
```

Comment:
Isn't the identifier typically "host" for extension cases?

Reply:
I used an example from a [real case](https://india.commcarehq.org/hq/admin/raw_doc/?id=6772cfb4-dee1-49be-a8fb-17376900307b) and that is how it was setup.